Documentation/docs.md

```diff
@@ -19,6 +19,7 @@ The easiest way to get started using etcd as a distributed key-value store for y
 Administrators who need to create reliable and scalable key-value stores for the developers they support should begin with a [cluster on multiple machines][clustering].
 
  - [Setting up clusters][clustering]
+ - [Docker guide][docker_guide]
```

Comment:
or probably we should say `Run etcd clusters instead containers` since you plan to add rkt.